justfile

```diff
@@ -0,0 +1,4 @@
+# Build a docker image locally and run it with docker-compose up
+docker-run:
```

Comment:
Maybe rename it to `build-and-run-docker`?

Reply:
done, thx for the suggestion